CHANGELOG.md

```diff
@@ -22,17 +22,21 @@ Given the `MAJOR.MINOR.PATCH` pattern, here is how we decide to increment:
 - Scroll to top functionality on footer
 - Added `/modules/util/web-storage-proxy.js` utility module.
 - Added `/modules/util/js-loader.js` utility module.
+- Adds ESLint check for `@todo` jsdoc syntax.
```

Comment:
You bring up the point though that entries shouldn't be deleted here. I wanted to consolidate some changes, but the earlier entry in the changelog could make it to beta before this PR does, so PRs shouldn't be deleting entries. That should happen in the refresh > beta PR or after. Added back deleted entry in https://github.com/cfpb/cfgov-refresh/commit/eaf534dc4700618422691cffcdfd9815452e288a